projects/js-packages/number-formatters/eslint.config.mjs

```diff
@@ -0,0 +1,27 @@
+// @ts-ignore
+import makeBaseConfig from 'jetpack-js-tools/eslintrc/base.mjs';
+
+export default [
+	...makeBaseConfig( import.meta.url ),
+	{
+		languageOptions: {
+			parserOptions: {
+				project: './tsconfig.json',
+			},
+		},
```

Comment:
Is this needed? I don't see it in any of our other TypeScript-using projects. The only one that has something like this is `projects/plugins/boost/eslint.config.mjs` because it has an extra `./tsconfig.eslint.json`.

P.S. You're welcome to do a `tsconfig.eslint.json` like Boost does if that's appropriate. It might be needed if you want tests linted properly?